microsoft/electionguard-api-python
# 📫 #222 postman election tests

### Issue

Fixes #222 

### Description
Adds postman tests for:

- manifest creation
- context creation
- election creation
- ballot encrypting
- ballot uploading

### Testing

1. Import the postman collection
2. Run it (e.g. click `End-To-End Happy Path` then "Run" then "Run ElectionGuard API Tests")

Expected all tests should pass.  It should look like this:

![image](https://user-images.githubusercontent.com/1769905/159018827-6ce8ea82-fdff-4645-9d7e-dc6db731d01c.png)
